mllib/src/main/scala/org/apache/spark/ml/stat/KolmogorovSmirnovTest.scala

```diff
@@ -81,32 +81,37 @@ object KolmogorovSmirnovTest {
    * Java-friendly version of `test(dataset: DataFrame, sampleCol: String, cdf: Double => Double)`
    */
   @Since("2.4.0")
-  def test(dataset: DataFrame, sampleCol: String,
-    cdf: Function[java.lang.Double, java.lang.Double]): DataFrame = {
-    val f: Double => Double = x => cdf.call(x)
-    test(dataset, sampleCol, f)
+  def test(
+      dataset: Dataset[_],
+      sampleCol: String,
+      cdf: Function[java.lang.Double, java.lang.Double]): DataFrame = {
+    test(dataset, sampleCol, (x: Double) => cdf.call(x))
```

Comment:
Seems like this broke Scala 2.12 build again. Can you fix it or shall I? Thanks!

Reply:
ah.... it's my fault. Sorry!

Reply:
[SPARK-23751][FOLLOW-UP] fix build for scala-2.12

Reply:
This is the 2nd time I've merged something which broke the Scala 2.12 build.  If we care about Scala 2.12, do you know if there are any efforts to add PR builder tests for 2.12?

Reply:
I can build locally against scala-2.12 to check it. :)